release_notes/unreleased.md

```diff
@@ -1 +1 @@
-**Unreleased**
```

Comment:
I'm not sure it matters either way, but you don't need to delete the header. I think it gets re-added to this file when the release notes are copied over for final release anyways

cc: @pzhou-splunk this is pretty minor but I think we can probably remove it from the autogeneration since we removed it for the other release note files

Reply:
Good to know, thanks!